samples/photo_album/README.md

```diff
@@ -0,0 +1,2229 @@
+This project was bootstrapped with [Create React App](https://github.com/facebookincubator/create-react-app).
```

Comment:
Add to the beginning of this generated README a description of the functionality of this sample and any other important information the reader might need